entrypoint.go

```diff
@@ -14,9 +14,11 @@ import (
 
 var (
 	endpoint = flag.String("cassy server endpoint", "localhost:20051", "endpoint of Cassy server")
+	mode     = flag.Bool("dev", false, "enable this option to run as dev server")
```

Comment:
Should this var name be more explicit like `devMode` or something like this ?

Reply:
Ok, I can change the variable name.